mobile-widgets/qmlmanager.cpp

```diff
@@ -456,7 +456,9 @@ void QMLManager::finishSetup()
 		openLocalThenRemote(url);
 	} else if (!empty_string(existing_filename) &&
 				qPrefCloudStorage::cloud_verification_status() != qPrefCloudStorage::CS_UNKNOWN) {
-		QMLPrefs::instance()->setCredentialStatus(qPrefCloudStorage::CS_NOCLOUD);
+		QMLPrefs::instance()->setOldStatus((qPrefCloudStorage::cloud_status)qPrefCloudStorage::cloud_verification_status());
+		set_filename(NOCLOUD_LOCALSTORAGE);
+		qPrefCloudStorage::set_cloud_verification_status(qPrefCloudStorage::CS_NOCLOUD);
```

Comment:
again, I struggle to correlate the commit message to the commit.
I can see that you are calling `setOldStatus()` and `set_filename()`, having that in the commit message is kinda pointless. What the commit message should tell me is why we are adding these calls.
And then the commit message talks about `saveCredentials()` which I cannot see in this commit. And it doesn't mention at all that we are switching from `setCredentialStatus()` to `set_cloud_verification_status()` and what the implications of that may be.

I'm not saying this code is wrong. I am just confused what you are doing here and the commit message is increasing that confusion.

Reply:
Refrased